go/avatars/fullcaching.go

```diff
@@ -257,7 +257,7 @@ func (c *FullCachingSource) populateCacheWorker(m libkb.MetaContext) {
 		found, ent, err := c.diskLRU.Get(m.Ctx(), m.G(), key)
 		if err != nil {
 			c.debug(m, "populateCacheWorker: failed to read previous entry in LRU: %s", err)
-			err := libkb.DiscardAndCloseBody(resp)
+			_, err := libkb.DiscardAndCloseBody(resp)
```

Comment:
This looks unmetered. Is that true or intentional? I'm not familiar with this, arg.url could be local.

Reply:
Ah good catch, missed a few of `libkb.ProxyHTTPGet`

Reply:
will update this in a subsequent PR, a larger change than expected